docs/Options.md

```diff
@@ -0,0 +1,530 @@
+| `SPACESHIP_EMBER_SUFFIX` | `$SPACESHIP_PROMPT_DEFAULT_SUFFIX` | Suffix after Ember.js section |
+| `SPACESHIP_EMBER_SYMBOL` | `🐹 ` | Character to be shown before Ember.js version |
+| `SPACESHIP_EMBER_COLOR` | `210` | Color of Ember.js section |
+
+### Kubectl context (`kubecontext`)
+
+Shows the active kubectl context.
+
+| Variable | Default | Meaning |
+| :------- | :-----: | ------- |
+| `SPACESHIP_KUBECONTEXT_SHOW` | `true` | Current Kubectl context section |
+| `SPACESHIP_KUBECONTEXT_PREFIX` | `at ` | Prefix before Kubectl context section |
+| `SPACESHIP_KUBECONTEXT_SUFFIX` | `$SPACESHIP_PROMPT_DEFAULT_SUFFIX` | Suffix after Kubectl context section |
+| `SPACESHIP_KUBECONTEXT_SYMBOL` | `☸️ ` | Character to be shown before Kubectl context |
+| `SPACESHIP_KUBECONTEXT_COLOR` | `cyan` | Color of Kubectl context section |
+
+### Execution time (`exec_time`)
+
+Execution time of the last command. Will be displayed if it exceeds the set threshold of time.
+
+| Variable | Default | Meaning |
+| :------- | :-----: | ------- |
+| `SPACESHIP_EXEC_TIME_SHOW` | `true` | Show execution time |
+| `SPACESHIP_EXEC_TIME_PREFIX` | `took ` | Prefix before execution time section |
+| `SPACESHIP_EXEC_TIME_SUFFIX` | `$SPACESHIP_PROMPT_DEFAULT_SUFFIX` | Suffix after execution time section |
+| `SPACESHIP_EXEC_TIME_COLOR` | `yellow` | Color of execution time section |
+| `SPACESHIP_EXEC_TIME_ELAPSED` | `2` | The minimum number of seconds for showing execution time section |
+
+### Battery (`battery`)
+
+By default, Battery section is shown only if battery level is below `SPACESHIP_BATTERY_THRESHOLD` (default: 10%) or it's fully charged.
+
+| Variable | Default | Meaning |
+| :------- | :-----: | ------- |
+| `SPACESHIP_BATTERY_SHOW` | `true` | Show battery section or not (`true`, `false`, `always` or `low`) |
+| `SPACESHIP_BATTERY_PREFIX` | `` | Prefix before battery section |
+| `SPACESHIP_BATTERY_SUFFIX` | `SPACESHIP_PROMPT_DEFAULT_SUFFIX` | Suffix after battery section |
+| `SPACESHIP_BATTERY_SYMBOL_CHARGING` | `⇡` | Character to be shown if battery is charging |
+| `SPACESHIP_BATTERY_SYMBOL_DISCHARGING` | `⇣` | Character to be shown if battery is discharging |
+| `SPACESHIP_BATTERY_SYMBOL_FULL` | `•` | Character to be shown if battery is full |
+| `SPACESHIP_BATTERY_THRESHOLD` | 10 | Battery level below which battery section will be shown |
+
+`SPACESHIP_BATTERY_SHOW` defines when to show battery section. Here are possible values:
+
+| `SPACESHIP_BATTERY_SHOW` | Below threshold | Above threshold | Fully charged |
+| :----------------------: | :-------------- | :-------------- | :------------ |
+| `false`                  | Hidden          | Hidden          | Hidden        |
+| `always`                 | Shown           | Shown           | Shown         |
+| `true`                   | Shown           | Hidden          | Shown         |
+| `low`                    | Shown           | Hidden          | Hidden        |
```

Comment:
This is definitely optional, but I must say, I wish all of the tables were formatted like this one, it's so much nicer to look at properly formatted tables like this one comparing to the tables like the one above. Even thought they are both rendered identically, I'm usually reviewing a non-rendered version, just because it's easier to immediately leave comments.

Many editors have some kind of "table formatter" extensions to do this automatically.

Reply:
I was just thinking about this and https://github.com/prettier/prettier does an awesome job formatting markdown documents. Have a look at https://github.com/salmanulfarzy/spaceship-zsh-theme/commit/57fa72f51a618de272072b1c3caa9e89a5b27b5f

Reply:
Thank you for this tip, definitely going to use it myself! 

Just a note, prettier renders columns containing \`\` incorrectly (see for example [Options.md:464](https://github.com/salmanulfarzy/spaceship-zsh-theme/commit/57fa72f51a618de272072b1c3caa9e89a5b27b5f#diff-b841bae586bf935661afa581b1afe307L464)), but I already reported this https://github.com/prettier/prettier/issues/3753

Reply:
I agree, but it's hard to maintain proper formatting for tables in long terms. Constant reformating to keep the same width for each row would kill git history related to these rows.

When I initially released Spaceship, I tried to maintain table formatting, but soon I realized that this doesn't play well for me.

Tools like Prettier would help much with maintaining formatting, but for now, it doesn't support shell scripts. I'd prefer to postpone adding Prettier right now.